### PR TITLE
[ADD] hr_timesheet_usability: Onchange that calculates the duration o…

### PR DESCRIPTION
**Pull request by Berezi**
…f the timesheets.

 A button in the tree view of the timesheets to put the end date.
 Two buttons in tasks to initiate and end a timesheet.

### CODE REVIEW

**Comment by Berezi** [comment]
@anajuaristi @oihane @Daniel-CA @alfredoavanzosc review please.

File: hr_timesheet_usability/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+{
+    "name": "Hr Timesheet Usability",
+    'version': '12.0.1.0.0',
+    "author": "Avanzosc",
+    "category": "Project",
+    "depends": [
+        "project",
+        "hr_timesheet",
+        "account"
+    ],
```
Comment (Daniel-CA) [inline hr_timesheet_usability/__manifest__.py:14]: "website": "http://www.avanzosc.es",

File: hr_timesheet_usability/models/account_analytic_line.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+class AccountAnalyticLine(models.Model):
+    _inherit = 'account.analytic.line'
+
+    date = fields.Datetime(string='Date')
+    date_end = fields.Datetime(string='Date end')
+
+    @api.multi
+    def _compute_duration(self):
+        self.ensure_one()
+        hours = 0
```
Comment (Daniel-CA) [inline hr_timesheet_usability/models/account_analytic_line.py]: hours = minutes = 0

File: hr_timesheet_usability/models/account_analytic_line.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+            hours = divmod(seconds, 3600)[0]
+            seconds = seconds - (hours * 3600)
+        if seconds >= 60:
+            minutes = divmod(seconds, 60)[0]
+            seconds = seconds - (minutes * 60)
+        return float('{}.{}'.format(hours, minutes))
+
+    @api.onchange("date", "date_end")
+    def onchange_dates(self):
+        if self.date:
```
Comment (Daniel-CA) [inline hr_timesheet_usability/models/account_analytic_line.py]: if self.date and self.date_end:
self.unit_amount = self._compute_duration()

Comment (Berezi) [reply hr_timesheet_usability/models/account_analytic_line.py]: Listo aldaketak eginak. Eskerrik asko.

File: hr_timesheet_usability/models/project_task.py [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+                if found:
+                    task.show_init_task = False
+                else:
+                    task.show_init_task = True
+
+    @api.multi
+    def action_button_initiate_task(self):
+        self.ensure_one()
+        timezone = pytz.timezone(self._context.get('tz') or 'UTC')
+        date_start = datetime.now()
```
Comment (oihane) [inline hr_timesheet_usability/models/project_task.py]: fields.Datetime.now()

**Comment by Berezi** [comment]
@oihane @Daniel-CA @anajuaristi @alfredoavanzosc review please.

**Review by alfredoavanzosc — APPROVED** [review]
:+1:

**Review by Daniel-CA — APPROVED** [review]

**Pull request merged**